admin/app/components/solidus_admin/sidebar/item/component.rb

```diff
@@ -10,9 +10,24 @@ def initialize(item:)
     @item = item
   end
 
+  # Arbitrary Tailwind background images is not working:
+  # https://github.com/tailwindlabs/tailwindcss/discussions/6617
   erb_template <<~ERB
-    <a href="#">
-      <%= @item.title %>
-    </a>
+    <div class="
+        py-2 px-3 mb-1
+        bg-no-repeat bg-left bg-origin-content
+      " style="background-image: url('<%= image_path(@item.icon) %>')">
```

Comment:
You think would be possible to embed the whole remix icon font and avoid adding the SVGs one by one? 

https://remixicon.com/download

Reply:
Thanks for pointing that out, it's a great improvement. Done!